client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -22,33 +22,100 @@ import {
 	getAutomatedTransferStatus,
 	isFetchingAutomatedTransferStatus,
 } from 'calypso/state/automated-transfer/selectors';
+import { getCurrentUser } from 'calypso/state/current-user/selectors';
 import { pluginInstallationStateChange } from 'calypso/state/marketplace/purchase-flow/actions';
 import { MARKETPLACE_ASYNC_PROCESS_STATUS } from 'calypso/state/marketplace/types';
 import { fetchSitePlugins } from 'calypso/state/plugins/installed/actions';
 import { getPluginsOnSite, isRequesting } from 'calypso/state/plugins/installed/selectors';
 import { fetchPluginData as wporgFetchPluginData } from 'calypso/state/plugins/wporg/actions';
 import { areFetched, getPlugins } from 'calypso/state/plugins/wporg/selectors';
 import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
-import siteHasFeature from 'calypso/state/selectors/site-has-feature';
-import { getSiteAdminUrl, isJetpackSite } from 'calypso/state/sites/selectors';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { ThankYouPluginSection } from './marketplace-thank-you-plugin-section';
 
 const ThankYouContainer = styled.div`
-	.marketplace-thank-you {
-		margin-top: 72px;
-		img {
-			height: 74px;
+	.thank-you__container {
+		padding-top: var( --masterbar-height );
+	}
+
+	.thank-you__container-header {
+		min-height: auto;
+		padding: 0px;
+		/* padding-top: 15px; */
+		padding-top: 60px;
+	}
+
+	.thank-you__body {
+		margin: 40px 0;
+
+		> div {
+			width: auto;
+			display: flex;
+			flex-direction: column;
+			align-items: center;
 		}
+
+		div {
+			min-width: auto;
+		}
+	}
+
+	.thank-you__step div > p {
+		padding: 0;
+		margin-bottom: 16px;
 	}
 
 	.thank-you__header-title {
-		font-size: 44px;
+		font-family: Recoleta;
+		font-size: 48px;
+		line-height: 24px;
```

Comment:
the title needs a little more space to the bottom before the subtitle.

Reply:
Updated here https://github.com/Automattic/wp-calypso/pull/71818/commits/a47d29c53b94ca2a74d5f17db32265ab8a1db5a9

The `line-height` property was the problem.